test/OpenTelemetry.Tests/Metrics/MetricApiTimerTest.cs

```diff
@@ -0,0 +1,59 @@
+// <copyright file="MetricApiTimerTest.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Collections.Generic;
+using System.Diagnostics.Metrics;
+using System.Threading.Tasks;
+using Xunit;
+
+#nullable enable
+
+namespace OpenTelemetry.Metrics.Tests
+{
+    public class MetricApiTimerTest
+    {
+        [Fact]
+        public void TestTimer()
+        {
+            using var meter = new Meter("BasicAllTest", "0.0.1");
+
+            var timer = meter.CreateTimer<int>("timer");
```

Comment:
What's the unit (in this PR it seems to imply milliseconds) of measure?
Do we allow people to change the unit (e.g. to nanoseconds, seconds)?

Reply:
Current code assumes milliseconds, as we create the underlying backing instrument with hardcoded unit of "ms". But, this is 100% open to anything the spec wants/needs.